tracker/labels.go

```diff
@@ -67,15 +88,29 @@ func (t *myTracker) RenameFirstTime(det objdet.Detection) objdet.Detection {
 	}
 	countLabel := baseLabel + "_" + strconv.Itoa(t.classCounter[baseLabel])
 	label := countLabel + "_" + GetTimestamp()
-	out := objdet.NewDetection(*det.BoundingBox(), det.Score(), label)
-	t.tracks[countLabel] = []objdet.Detection{out} // Start new track with this one
+	out := ReplaceLabel(det, label)
+	// start a new track, but it will be tentative, and may be removed if lost
+	// before persistence counter reaches "stable"
+	t.tracks[countLabel] = []*track{out}
 	return out
 }
 
-func (t *myTracker) UpdateTrack(det objdet.Detection) {
-	countLabel := strings.Join(strings.Split(det.Label(), "_")[0:2], "_")
-	track, ok := t.tracks[countLabel]
+func getTrackingLabel(tr *track) string {
```

Comment:
since i used this somewhere else now too (to delete dropped tracks) I put this into its own function